src/base/types/CoreModel.jl

```diff
@@ -93,6 +93,26 @@ function objective(a::CoreModel)::SparseVec
     a.c
 end
 
+"""
+    reaction_equation(model::CoreModel, rxn_id::String)::Dict{String, Float64}
+
+Return the reaction equation of reaction with id `rxn_id` in model. The reaction
+equation maps metabolite ids to their stoichiometric coefficients.
+"""
+function reaction_equation(m::CoreModel, rxn_id::String)::Dict{String, Float64}
```

Comment:
this is precisely called "reaction stoichiometry" in SBML,  I'd suggest we use the same naming.

Also, can we add this to the abstract accesors?